test/test-clusters/src/main/java/org/elasticsearch/test/cluster/local/LocalClusterFactory.java

```diff
@@ -171,15 +171,25 @@ public String getTransportEndpoint() {
             return readPortsFile(portsFile).get(0);
         }
 
+        public String getRemoteClusterServerEndpoint() {
+            Path portsFile = workingDir.resolve("logs").resolve("remote_cluster.ports");
```

Comment:
Just for my education as I'm not familiar with this. So this remote cluster feature actually uses a unique transport separate from the normal transport address?

Reply:
Yes. We had some back and forth on the decision. But yes the new remote cluster feature has its own separate port. It still speaks the transport protocol.